docker/config/test.env

```diff
@@ -0,0 +1 @@
+DB_NAME=test_location
```

Comment:
This covers getting the tests to run against a different db. We might want to do the same thing for redis, but it kind of looks like all the redis usage is mocked out.

Reply:
When I try ``make testshell``, and then ``pytest``, the tests get stuck. The "right" way to run tests is ``TESTING=true pytest``. We could add a line to this file so that just pytest works:

```
TESTING=true
```

``make shell`` doesn't load this file, so ``TESTING`` will remain unset.

Reply:
Oops--that's a mistake. Good catch!

Also, we should probably do a pass on removing/reducing uses of `TESTING` so as to reduce complexity.